saleor/graphql/core/filters.py

```diff
@@ -70,6 +76,14 @@ def filter_status(qs, _, value):
     return qs.filter(status=value)
 
 
+def filter_metadata(qs, _, value):
+    for metadata_item in value:
+        lookup = "__".join(["metadata", metadata_item.key])
```

Comment:
This is correct, but I would suggest using an f-string for better readability:
```suggestion
        lookup = f'metadata__{metadata_item.key}'
```

Reply:
@maarcingebala I've used fstrings at the beginning but I said to myself I don't know what's the least python version you're supporting, so I chose the one that's more supported, I'll change it for sure!

Reply:
And thanks for the hints, I'll do them, can you have a look at #6684